ansible/roles/master/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 kube_master_insecure_port: 8080
 
-localBuildOutput: ../../_output/local/go/bin
\ No newline at end of file
+localBuildOutput: ../../_output/local/go/bin # localBuildOutput: ../../kubernetes/_output/dockerized/bin/linux/amd64/ (docker builds)
```

Comment:
Should this not default to `../../kubernetes/.....`    instead of just `../../_output`?